content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -261,21 +261,35 @@ problems, the kubelet resets the restart backoff timer for that container.
 [Sidecar containers and Pod lifecycle](/docs/concepts/workloads/pods/sidecar-containers/#sidecar-containers-and-pod-lifecycle)
 explains the behaviour of `init containers` when specify `restartpolicy` field on it.
 
+### Reduced container restart delay
+
+{{< feature-state
+feature_gate_name="ReduceDefaultCrashLoopBackOffDecay" >}}
+
+With the alpha feature gate `ReduceDefaultCrashLoopBackOffDecay` enabled,
```

Comment:
nit: we'd like this to be a default, right? I would suggest to put a note in the paragraph above that the numbers listed are only applicable when the feature gate is disabled. It may be too early for alpha, but when it goes to beta we will for sure switch numbers in paragraph above and change this section to "Old restart delay numbers".